packages/components/src/drop-zone/style.scss

```diff
@@ -7,39 +7,33 @@
 	z-index: z-index(".components-drop-zone");
 	visibility: hidden;
 	opacity: 0;
-	transition: 0.3s opacity, 0.3s background-color, 0s visibility 0.3s;
-	@include reduce-motion("transition");
-	border: 2px solid var(--wp-admin-theme-color);
 	border-radius: 2px;
 
 	&.is-active {
 		opacity: 1;
 		visibility: visible;
-		transition: 0.3s opacity, 0.3s background-color;
-		@include reduce-motion("transition");
 	}
 
-	&.is-dragging-over-element {
-		background-color: var(--wp-admin-theme-color);
-	}
+	// &.is-dragging-over-element {
```

Comment:
Comments here can be removed. I understand that the blue sheet now animates to become an overlay, rather than just change the background color.